test/screens/v2/candidate_generator/elevator/closures_test.exs

```diff
@@ -71,14 +71,6 @@ defmodule Screens.V2.CandidateGenerator.Elevator.ClosuresTest do
   end
 
   describe "header and footer" do
-    test "have no variant when current elevator is not closed", %{now: now} do
-      assert [
-               _elevator_closures,
-               %NormalHeader{screen: @screen, text: "Elevator 111", time: ^now, variant: nil},
-               %Footer{screen: @screen, variant: nil}
-             ] = Generator.elevator_status_instances(@screen, now)
-    end
-
```

Comment:
I think this test case still covers a unique and relevant behavior: that the footer is present (with no variant) when the current elevator is not closed but there are active closures elsewhere. Rather than deleting it, we could add some setup that creates an active closure, so the footer is still generated.